recipes/protobuf/all/conanfile.py

```diff
@@ -48,7 +48,7 @@ def configure(self):
                                                 "Visual Studio 2015 or higher.")
 
     def requirements(self):
-        if self.options.with_zlib:
+        if self.options.with_zlib and not self.options.lite:
```

Comment:
Should the `self.options.with_zlib` option be removed if `options.lite == True` in `configure`?

Reply:
Yes, makes more sense. zlib is required (as optional) by libprotobuf only.